.eslintrc

```diff
@@ -1,12 +1,16 @@
 plugins:
   - fxa
+
 extends: plugin:fxa/server
 
 rules:
+  complexity: 0
   handle-callback-err: 0
   strict: 2
-  semi: 0
+  semi: ["error", "always"]
```

Comment:
This line can just be removed, since we pick it up from `plugin:fxa/server`

Reply:
I don't know why...but it is not picking this line from there.

Reply:
saying this because all the semicolon changes disappeared from the previous commit as soon as I run this command `eslint lib scripts test --fix`.